src/plugins/cams/datasource.py

```
@@ -0,0 +1,509 @@
+                measurement_count += 1
+                
+                # Yield in batches
+                if len(measurements) >= 100:
+                    yield measurements
+                    measurements = []
+                    
+            # Yield remaining measurements
+            if measurements:
+                yield measurements
+                
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)